backend/game/turn_actions/test_player_movement.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+
+from game.player.player import Player
+import game.turn_actions.player_movement as player_movement
+from game.values.locations import Locations
+from game.dice.dice_resolver import dice_resolution
+from game.dice.dice import DieValue
+from game.values.constants import DEFAULT_HEALTH
+
+MOCK_DICE_VALUES = [DieValue.ATTACK, DieValue.ATTACK, DieValue.HEAL, DieValue.HEAL, DieValue.HEAL, DieValue.HEAL]
+
+
+@pytest.fixture(autouse=True)
+def main_player():
+    player_main = Player()
+    player_main.location = Locations.OUTSIDE
+    return player_main
+
+
+@pytest.fixture(autouse=True)
+def other_players():
+    player_a = Player()
+    player_b = Player()
+    player_c = Player()
+    players = [player_a, player_b, player_c]
+    for player in players:
+        player.location = Locations.OUTSIDE
+    return players
+
+
+def test_move_to_empty_tokyo(main_player, other_players):
```

Comment:
EDIT The test I was looking for was down below

Reply:
Assuming you mean test_move_to_empty_tokyo_gives_victory_point